python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -1897,3 +1910,74 @@ def unique_id_from_asset_and_check_keys(entity_keys: Iterable["EntityKey"]) -> s
     """
     sorted_key_strs = sorted(str(key) for key in entity_keys)
     return non_secure_md5_hash_str(json.dumps(sorted_key_strs).encode("utf-8"))[:8]
+
+
+def _thread_deps_to_node(
```

Comment:
function name a bit add.

`_assets_def_with_additional_deps` or something like that more clear

Reply:
changed the name, also should reflect the fact that we're potentially deleting deps now too